server/api/alembic/seeds/types.csv

```diff
@@ -8,6 +8,6 @@ type_id,type_name,agency_id,color,description,data_code
 7,Metal/Appliances,2,#8B508B,"Air conditioners, dryers, refrigerator and more...",Metal/Household Appliances
 8,Single Streetlight,1,#79B74E,"Pole knocked down, streetlight outage on a wooden power pole, or malfunctioning traffic signal",Single Streetlight Issue
 9,Multiple Streetlights,1,#EDAD08,"Multiple poles knocked down, streetlight outages on wooden power poles, or malfunctioning traffic signals",Multiple Streetlight Issue
-10,Water Waste,1,#EDAD08,"Water runoff, over-watering, incorrect water days, or any other water waste ",Report Water Waste
+10,Water Waste,1,#DCED08,"Water runoff, over-watering, incorrect water days, or any other water waste ",Report Water Waste
```

Comment:
Revert this change since it's part of #1226

Reply:
Resolve